application-workloads/azure-gamedev/gamedev-vm/scripts/PostInstall.ps1

```diff
@@ -0,0 +1,1021 @@
+        $GPO = $gptstring -match "gPCMachineExtensionNames"
+        $add = '[{42B5FAAE-6536-11D2-AE5A-0000F87571E3}{40B6664F-4972-11D1-A7CA-0000F87571E3}]'
+        $replace = "$GPO" + "$add"
+        (Get-Content "C:\Windows\System32\GroupPolicy\gpt.ini").Replace("$GPO","$replace") | Set-Content "C:\Windows\System32\GroupPolicy\gpt.ini"
+        [int]$i = $gpoversion.trim("Version=") 
+        [int]$n = $gpoversion.trim("Version=")
+        $n +=2
+        (Get-Content C:\Windows\System32\GroupPolicy\gpt.ini) -replace "Version=$i", "Version=$n" | Set-Content C:\Windows\System32\GroupPolicy\gpt.ini
+        }
+    else{
+        write-output "Not Required"
+        }
+    }
+
+#Adds Premade Group Policu Item if existing configuration doesn't exist
+function addRegItems{
+    ProgressWriter -Status "Adding Registry Items and Group Policy" -PercentComplete $PercentComplete
+    if (Test-Path ("C:\Windows\system32\GroupPolicy" + "\gpt.ini")) {
+        add-gpo-modifications
+        }
+    Else {
+        Move-Item -Path $path\ParsecTemp\PreInstall\gpt.ini -Destination C:\Windows\system32\GroupPolicy -Force | Out-Null
+        }
+    regedit /s $path\ParsecTemp\PreInstall\NetworkRestore.reg
+    regedit /s $path\ParsecTemp\PreInstall\ForceCloseShutDown.reg
+    New-PSDrive -PSProvider Registry -Name HKU -Root HKEY_USERS -ErrorAction SilentlyContinue | Out-Null
+    }
+
+function Test-RegistryValue {
+    # https://www.jonathanmedd.net/2014/02/testing-for-the-presence-of-a-registry-key-and-value.html
+    param (
+
+     [parameter(Mandatory=$true)]
+     [ValidateNotNullOrEmpty()]$Path,
+
+    [parameter(Mandatory=$true)]
+     [ValidateNotNullOrEmpty()]$Value
+    )
+
+    try {
+        Get-ItemProperty -Path $Path | Select-Object -ExpandProperty $Value -ErrorAction Stop | Out-Null
+        return $true
+        }
+    catch {
+        return $false
+        }
+
+}
+
+
+#Create ParsecTemp folder in C Drive
+function create-directories {
+    ProgressWriter -Status "Creating Directories (C:\ParsecTemp)" -PercentComplete $PercentComplete
+    if((Test-Path -Path C:\ParsecTemp) -eq $true) {} Else {New-Item -Path C:\ParsecTemp -ItemType directory | Out-Null}
+    if((Test-Path -Path C:\ParsecTemp\Apps) -eq $true) {} Else {New-Item -Path C:\ParsecTemp\Apps -ItemType directory | Out-Null}
+    if((Test-Path -Path C:\ParsecTemp\DirectX) -eq $true) {} Else {New-Item -Path C:\ParsecTemp\DirectX -ItemType directory | Out-Null}
+    if((Test-Path -Path C:\ParsecTemp\Drivers) -eq $true) {} Else {New-Item -Path C:\ParsecTemp\Drivers -ItemType Directory | Out-Null}
+    # if((Test-Path -Path C:\ParsecTemp\Devcon) -eq $true) {} Else {New-Item -Path C:\ParsecTemp\Devcon -ItemType Directory | Out-Null}
+    }
+
+    #Create ParsecTemp folder in C Drive
+function unzip-preinstall {
+    ProgressWriter -Status "Creating Directories (C:\ParsecTemp)" -PercentComplete $PercentComplete
+    Copy-Item ".\scripts" "C:\ParsecTemp\PreInstall"
```

Comment:
need to confirm with team that popping the zip into the directory has been updated correctly.

Reply:
I think you want to copy the "parsec" folder instead of "scripts".

Reply:
```suggestion
    Copy-Item ".\parsec" "C:\ParsecTemp\PreInstall"
```